Makefile

```diff
@@ -27,6 +27,10 @@ livehtml:
 livehtmlall:
 	sphinx-autobuild -a "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O) --watch _static
 
+linkcheck-changed-files:
```

Comment:
I know this is nitpicking, but could we maybe call this target something more like `linkcheck-specified-files`, since if I do `make linkcheck-changed-files` at the command line, it still checks all the files?

Reply:
Hi @TibsAtWork this command can only check **.rst** files, but yeah, I track the changed files and run a link check.

Reply:
Honestly, you are much better than me with the naming. So yeah, I will change ^^